src/swap/SwapScreen.tsx

```diff
@@ -215,12 +215,7 @@ export function SwapScreenSection({ showDrawerTopNav }: { showDrawerTopNav: bool
   const handleShowTokenSelect = (fieldType: Field) => () => {
     ValoraAnalytics.track(SwapEvents.swap_screen_select_token, { fieldType })
     // ensure that the keyboard is dismissed before animating token bottom sheet
-    Keyboard.addListener('keyboardDidHide', onKeyboardDidHide(fieldType))
     Keyboard.dismiss()
-  }
-
-  const onKeyboardDidHide: any = (fieldType: Field) => {
-    Keyboard.removeListener('keyboardDidHide', onKeyboardDidHide)
```

Comment:
`removeListener` no longer exists on `Keyboard`

Reply:
I assume it's because `addListener` returns a callback to stop listening, right?
Don't we need to keep the initial logic?

Reply:
yes that's right. i opted for this logic change because idk how to reference the return value of `addListener` inside the callback of the `addListener`. i'm not sure that the existing logic is adding value anyway, i've attached videos of what i see on the nightly vs what i see on this branch. they look similar enough to me (the nightly build keyboard is not completely dismissed before the bottom sheet starts animating)

nightly:

https://github.com/valora-inc/wallet/assets/20150449/c735d466-6e13-4f00-9bcd-643ead6b9e04


this branch:

https://github.com/valora-inc/wallet/assets/20150449/e7cb9fac-3047-49fd-b661-c93a331cef29

Reply:
Great, thanks for double checking, I don't see much difference either.